### PR TITLE
Rename tutorials

### LINKED ISSUE #104: Removed "intermediates" and "advanced" tutorials name
## Intuition
1. "intermediates" isn't a word

We can just have in the "api call" tutorial that the randomness one is a prerequisite. 

https://docs.chain.link/docs/intermediates-tutorial/

### PR DESCRIPTION
**Pull request by dmitrydao**
fixes #104

### CODE REVIEW

**Review by PatrickAlphaC — CHANGES_REQUESTED** [review]
All the URLs now start with `-`, please address. Let's keep the URLs as they are for now so we don't break anything.

Names are fine. Can we leave on "the basics" on the basics tutorial

File: docs/Introduction/tutorials/beginners-tutorial.md [hunk 1]
```diff
@@ -5,10 +5,9 @@ date: Last Modified
 title: "Beginners - The Basics"
 permalink: "docs/beginners-tutorial/"
 excerpt: "Smart Contracts and Chainlink"
-whatsnext: {"Get the Latest Price":"/docs/get-the-latest-price/", "Deploy your first contract":"/docs/deploy-your-first-contract/", "Intermediates - Random Numbers":"/docs/intermediates-tutorial/"}
-hidden: false
+whatsnext: {"Get the Latest Price":"/docs/get-the-latest-price/", "Deploy your first contract":"/docs/deploy-your-first-contract/", "Random Numbers Tutorial":"/docs/intermediates-tutorial/"}
 metadata: 
-  title: "Beginners Tutorial"
+  title: "Beginners The Basics"
```
Comment (StephenFluin) [inline docs/Introduction/tutorials/beginners-tutorial.md:10]: This title doesn't make sense to me? 'Beginners The Basics'?

**Review by StephenFluin — APPROVED** [review]

**Comment by StephenFluin** [comment]
@PatrickAlphaC Need your approval to merge.

**Review by PatrickAlphaC — APPROVED** [review]

**Comment by PatrickAlphaC** [comment]
LGTM

**Pull request merged**